### DISCUSSION TITLE
How to reuse spring config on multiple images? Ref: UseSpring-3D Card

### DISCUSSION THREAD

**Discussion by konnerb** [pmndrs/react-spring#996] [Support] [answered]
Hi everyone,

How can you reuse this [UseSpring - 3D Card](https://codesandbox.io/embed/rj998k4vmm) example on multiple images?

I've tried implementing the same config on UseSprings and tested react-use-gesture methods, but still haven't made any progress..

If there are any examples recommended, that would be more than enough!

Thanks!

**Comment by konnerb** [comment, marked as answer]
Figured out how to animate multiple images.

Here's the link for anyone interested: https://codesandbox.io/s/use-springsmultiple-images-rnf6t

And if there's a better way of doing this, please reach out!

**Reply by bittu-choudhary** [reply]
Thanks, @konnerb. Coincidently, I was trying something similar with diff category cards. Your example hit the bull's eye.

**Reply by konnerb** [reply]
Glad it was of help!
